serialization/api.go

```diff
@@ -155,11 +162,47 @@ type DataOutput interface {
 	// WriteStringArray writes a []string in UTF-8 format.
 	WriteStringArray(v []string)
 
-	// WriteBytes writes a string's characters.
+	// WriteStringBytes writes a string's characters.
 	WriteStringBytes(bytes string)
 
 	// WriteZeroBytes writes zero bytes as given length.
 	WriteZeroBytes(count int)
+
+	// WriteDate writes the given date.
+	WriteDate(t time.Time)
```

Comment:
These methods do not exist in the DataOutput interface in Java

Reply:
Removed them at: https://github.com/hazelcast/hazelcast-go-client/pull/695/commits/3f2603613904c066ed58d86749debf6358fe5cec